packages/firestore/lite/index.d.ts

```diff
@@ -45,6 +45,7 @@ export function setLogLevel(logLevel: LogLevel): void;
 
 export interface FirestoreDataConverter<T> {
   toFirestore(modelObject: T): DocumentData;
+  toFirestore(modelObject: Partial<T>, options: SetOptions): DocumentData;
   fromFirestore(snapshot: QueryDocumentSnapshot): T;
```

Comment:
Should we explicitly type this as QueryDocumentSnapshot<DocumentData> to make it easier to understand?

Reply:
done.